src/components/MovieCard/MovieCard.js

```diff
@@ -11,10 +11,15 @@ export const MovieCard = ({
   <div className="card">
     <div className="card-image">
       <figure className="image is-4by3">
-        <img
-          src={imgUrl}
-          alt="Film logo"
-        />
+        {imgUrl === 'N/A'
+          ? (<p className="image__empty">Sorry, there is no poster</p>)
+          : (
+            <img
+              src={imgUrl}
+              alt="Film logo"
+            />
+          )
```

Comment:
```suggestion
          : <img src={imgUrl} alt="Film logo" />
```
когда атрибута меньше 3ех, пишем в 1 строку